### ISSUE TITLE
 Translate and review English subtitle of 'RChain Architecture & Implementation detailed' video from the Rchain Youtube Channel into chinese

### ISSUE THREAD

**Issue by linsheng9731** [rchain/bounties#438]
I wanne translate and review English subtitle of [RChain Architecture & Implementation detailed](https://www.youtube.com/watch?v=bwH66UMFp-Y&list=PLf2bbiic5ZjCPzin3gCSMGiBtbT8UO5o2&index=10) video from the Rchain Youtube Channel into chinese. This video really helped me understand the Rchain Architecture.

Measure of Completion:  
Suggested Budget: $2382(0.4$/word)

  - Why this document? (mark all that apply)
    - [x] it's one of the /rchain/reference documents
  - How can people who do not speak this language be assured of the quality of the translation? (all are required)
    - [x] several reviewers are involved
    - [x] at least two of them have high proficiency in (the target language)
    - [x] at least two of them have high proficiency in English
    - [x] at least two of them have either
       - [x] significant reputation (e.g. a diploma) at stake; master degree of CS
       - [x] a track record of quality work on rchain bounties: #242, #176, ...
  - What about maintenance? (choose one)
    - [x] the source document is stable; further translation work is not expected

**Comment by linsheng9731** [comment]
Hi all, we have finished translate work, thanks for @rjl493456442 @AbnerZheng @yuhuanmocmoc ! And  we have merged the subtitle into video. Further more, we uploaded the video to [Youku](http://v.youku.com/v_show/id_XMzQ5OTMwNDI0OA) which is largest video share website in China. And this video also will be post to Rchain China wechat account "Rchain Blackboard", let more people know the great work of rchain team!
